src/feditest/templates/test_matrix.jinja2

```diff
@@ -1,35 +1,63 @@
 
-<table class="tests">
-    <thead>
-        <tr>
-            <th>Test</th>
-            {%- for run_session, plan_session in sessions %}
-            <th>
-                {%- if session_links -%}
-                <a href="#{{ run_session.name }}">{{ run_session.name }}</a>
-                {%- else -%}
-                {{ run_session.name }}
-                {%- endif -%}
-            </th>
-            {%- endfor %}
-        </tr>
-    </thead>
-    <tbody>
-        {% for test_spec in all_tests -%}
-        <tr>
-            <td class="name">{{ test_spec.name }}</td>
-            {%- for run_session, plan_session in sessions -%}
-                {% set problem = get_problem(run_session, test_spec) -%}
-                {% if test_spec.disabled -%}
-                    {% set status = "skipped" -%}
-                {% elif problem -%}
-                    {% set status = "failed" -%}
-                {% else -%}
-                    {% set status = "passed" -%}
-                {% endif %}
-            <td class="status {{ status }}">{{ status }}</td>
-            {%- endfor %}
-        </tr>   
-        {%- endfor %}
-    </tbody>
-</table>
+    <div class="tests">
+        <table class="tests">
+            <colgroup>
+                <col>
+{%- for run_session, plan_session in sessions %}
```

Comment:
Just curious... do you prefer this template formatting or was this done automatically by something?

Reply:
HTML templates tend to have these two hierarchies intermingled -- HTML and code. I have a hard time "seeing" the structure of either if they are indented as one, and then the emitted HTML has strange indentation, too. Easier for me to read if they are indented separately. Just personal preference, we don't have to do it this way.